admin/bin/qserv-admin.py

```diff
@@ -154,7 +186,7 @@ def receiveCommands(self):
         cmd = ''
         prompt = self._prompt
         while True:
-            line = raw_input(prompt).decode("utf-8").strip()
+            line = raw_input(prompt).strip()
```

Comment:
The unicode problem is somewhat bigger that I'm prepared to handle here. I removed `decode()` because new CSS interface is SWIGed C++ code and it does not handle unicode at all. Previous implementation was using kazoo which required unicode. For now we'll be passing bytes directly to C++ and hope that no one uses anything outside ASCII and no fancy encodings like utf16. I'm not sure how it's going to work for Python3 (I mean SWIG).